extensions/podman/package.json

```diff
@@ -113,7 +113,8 @@
           "minimum": 1,
           "maximum": "HOST_TOTAL_CPU",
           "scope": "ContainerProviderConnectionFactory",
-          "description": "CPU(s)"
+          "description": "CPU(s)",
+          "when": "!isWindows || !podman.isWSL"
```

Comment:
do we need to replicate in 3 different places the condition or we introduce direct flags like
```
"when": "podmanMachineCpuSupported"
"when": "podmanMachineDiskSupported"
"when": "podmanMachineMemorySupported"
```

Reply:
I don't get what you're saying. It is already replicated in three places. Do you prefer direct flags?

Reply:
I mean, it was to avoid such complex conditions as we handle both the package.json and the condition in our case.

so I don't know if we need to put all the logic inside the package.json file or just make it working in the code (as it's more tested)